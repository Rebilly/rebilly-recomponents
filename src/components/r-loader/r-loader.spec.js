import {shallowMount} from '@vue/test-utils';
import {renderToString} from '@vue/server-test-utils';
import RLoader from './r-loader.vue';

describe('r-loader.vue', () => {
    it('renders props.msg when passed', () => {
        const wrapper = shallowMount(RLoader);
        expect(wrapper).toMatchSnapshot();
    });

    it('should render via SSR and match snapshot', async () => {
        const wrapper = renderToString(RLoader);
        expect(wrapper).toMatchSnapshot();
    });
});
