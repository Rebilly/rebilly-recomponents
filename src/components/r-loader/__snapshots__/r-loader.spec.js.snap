// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`r-loader.vue renders props.msg when passed 1`] = ``;

exports[`r-loader.vue should render via SSR and match snapshot 1`] = `<!---->`;
